scripts/perf/deploy_horicromtal/deploy_horicromtal.sh

```diff
@@ -0,0 +1,96 @@
+
+runner_metadata=(
+  "BUILD_NUMBER=$BUILD_NUMBER"
+  "BUILD_TAG=$BUILD_TAG"
+  "CLEAN_UP=false"
+  "CLOUD_SQL_INSTANCE=$CLOUD_SQL_INSTANCE"
+  "CROMWELL_DB_USER=cromwell"
+  "CROMWELL_DB_PASS=$DB_PASS"
+  "CROMWELL_DOCKER_IMAGE=$CROMWELL_DOCKER_IMAGE"
+  "CROMWELL_PROJECT=broad-dsde-cromwell-perf"
+  "CROMWELL_BUCKET=$GCS_BUCKET"
+  "CROMWELL_STATSD_HOST=10.128.0.4"
+  "CROMWELL_STATSD_PORT=8125"
+  "CROMWELL_PERF_SCRIPTS_BRANCH=${REPO_BRANCH}"
+  "GCS_REPORT_PATH=${GCS_REPORT_PATH}"
+  "GCS_REPORT_BUCKET=${GCS_REPORT_BUCKET}"
+  "TEST_CASE_DIRECTORY=${TEST_CASE_DIRECTORY}"
+)
+
+reader_metadata=(
+  "BUILD_NUMBER=$BUILD_NUMBER"
+  "BUILD_TAG=$BUILD_TAG"
+  "CLEAN_UP=false"
+  "CLOUD_SQL_INSTANCE=$CLOUD_SQL_INSTANCE"
+  "CROMWELL_DB_USER=cromwell"
+  "CROMWELL_DB_PASS=$DB_PASS"
+  "CROMWELL_DOCKER_IMAGE=$CROMWELL_DOCKER_IMAGE"
+  "CROMWELL_PROJECT=broad-dsde-cromwell-perf"
+  "CROMWELL_BUCKET=$GCS_BUCKET"
+  "CROMWELL_STATSD_HOST=10.128.0.4"
+  "CROMWELL_STATSD_PORT=8125"
+  "CROMWELL_PERF_SCRIPTS_BRANCH=${REPO_BRANCH}"
+  "GCS_REPORT_PATH=${GCS_REPORT_PATH}"
+  "GCS_REPORT_BUCKET=${GCS_REPORT_BUCKET}"
+  "TEST_CASE_DIRECTORY=reader"
+)
+
+cp scripts/perf/deploy_instance/run_on_instance.sh mnt/
+
+gcloud_deploy_instance \
+  "perf_deploy_instance_${BUILD_NUMBER}-runner101" \
+   "${CROMWELL_INSTANCE_NAME}-runner101" \
+    "${INSTANCE_TEMPLATE}" \
+     "$(join ${runner_metadata[@]})" | tee runner1.txt
+
+gcloud_deploy_instance \
+  "perf_deploy_instance_${BUILD_NUMBER}-runner102" \
+   "${CROMWELL_INSTANCE_NAME}-runner102" \
+   "${INSTANCE_TEMPLATE}" \
+   "$(join ${runner_metadata[@]})" | tee runner2.txt
+
+gcloud_deploy_instance \
+  "perf_deploy_instance_${BUILD_NUMBER}-reader101" \
+   "${CROMWELL_INSTANCE_NAME}-reader101" \
+   "${INSTANCE_TEMPLATE}" \
+   "$(join ${reader_metadata[@]})" | tee reader1.txt
+
+AWK_STRING_PREFIX='{print$'
+AWK_STRING_SUFFIX='}'
+AWK_STRING="${AWK_STRING_PREFIX}${FIELD_NO_TO_RETURN}${AWK_STRING_SUFFIX}"
+
+typeset RUNNER_UNDER_TEST_1=$(cat runner1.txt | tail -n1 | awk '{print $5}')
+typeset RUNNER_UNDER_TEST_2=$(cat runner2.txt | tail -n1 | awk '{print $5}')
+
+typeset READER_UNDER_TEST_1=$(cat reader1.txt | tail -n1 | awk '{print $5}')
+typeset CROMWELL_VALUE_TO_RETURN=$(cat reader1.txt | tail -n1 | awk ${AWK_STRING} )
+
+if test -z "${RUNNER_UNDER_TEST_1}" -o -z "${RUNNER_UNDER_TEST_2}" -o -z "${READER_UNDER_TEST_1}"
+then
+  echo "One of RUNNER_UNDER_TEST_1 or RUNNER_UNDER_TEST_2 or READER_UNDER_TEST_1 are empty ('${RUNNER_UNDER_TEST_1}', '${RUNNER_UNDER_TEST_2}, ${READER_UNDER_TEST_1}')"
+  exit 1
+else
+  echo "Determined that RUNNER_UNDER_TEST_1=${RUNNER_UNDER_TEST_1}, RUNNER_UNDER_TEST_2=${RUNNER_UNDER_TEST_2}, READER_UNDER_TEST_1=${READER_UNDER_TEST_1}"
+fi
+
+CROMWELL_UNDER_TEST="${RUNNER_UNDER_TEST_1}"
+custom_wait_for_cromwell
+
+CROMWELL_UNDER_TEST="${RUNNER_UNDER_TEST_2}"
+custom_wait_for_cromwell
+
+CROMWELL_UNDER_TEST="${READER_UNDER_TEST_1}"
+custom_wait_for_cromwell
```

Comment:
TOL It seems strange that this variable has to be serially reassigned rather than passing it as a parameter to the function